src/content/advanced/storage/azure-pvc-encryption/index.md

```diff
@@ -0,0 +1,79 @@
+---
+linkTitle: Persistent Volumes data encryption with Azure Key Vault
+title: Encrypting Persistent Volumes using Azure Key Vault
+description: Tutorial on how to encrypt dynamically provisioned Persistent Volumes on Azure clusters using Azure Key Vault feature.
+weight: 40
+menu:
+  main:
+    parent: advanced-storage
+user_questions:
+  - How do I encrypt data on Persistent Volumes?
+  - How do I secure data on Persistent Volumes?
+  - How do I use Azure Key Vault?
+aliases:
+  - /guides/encrypting-persistent-volumes-on-azure-with-azure-key-vault/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-celestial
+---
+
+# Using Persistent Volumes on Azure
```

Comment:
The first headline should be semantically the same as the title. The title sounds much more specific than this headline.

Reply:
opened up new PR: https://github.com/giantswarm/docs/pull/984